components/BadgeCounter/src/index.tsx

```diff
@@ -14,9 +13,7 @@ export const BadgeCounter: React.FC<BadgeCounterProps> = (props: BadgeCounterPro
   const rootClassNames = clsx('denhaag-badge-counter', props.className);
   return (
     <div id={props.id} className={rootClassNames}>
-      <DotIndicator overlap={'rectangle'}>
```

Comment:
Ik had gemist dat de DotIndicator hier nog voor gebruikt werd. Zie dat in de templates de BadgeCounter van Utrecht gebruikt wordt. Mogelijk kunnen we die van Den Haag uitfaseren, en overal die van Utrecht gaan gebruiken. Dan hebben we ook geen last van deze wijziging.